### ISSUE TITLE
Tree find widget does not have border

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#175951]
Type: <b>Bug</b>

1. Enable high contrast dark them
2. In explorer, start find

**Bug**
Unlike the editor find widget, the tree find widget does not get a border:

![Screenshot 2023-03-02 at 11 10 21 AM](https://user-images.githubusercontent.com/12821956/222528132-c7af2592-7b29-44d5-8150-26d2d67f3ed5.png)

This makes it harder to see

VS Code version: Code - Insiders 1.77.0-insider (Universal) (155cd6db223feb1f8c734ca70e108b9efec7b054, 2023-03-02T12:44:52.544Z)
OS version: Darwin x64 22.3.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 4, 4|
|Memory (System)|32.00GB (0.01GB free)|
|Process Argv|--crash-reporter-id 48781ca2-1705-4f64-9bab-325055aab55d|
|Screen Reader|no|
|VM|0%|
</details>
